### PR TITLE
CLUSTERMAN-637: Add basic Rookout support

### PR DESCRIPTION
**Pull request by BeryJu**
### Description

COMPINFRA-1009, adding it to clusterman first as there's a bunch of tooling and setup already done that makes it easier to add rookout to paasta services, compared to other projects

Rookout is disabled by default so this should be safe to deploy, can be enabled at any time from soaconfigs

### CODE REVIEW

File: requirements-minimal.txt [hunk 1]
```diff
@@ -19,3 +19,7 @@ signalfx
 simplejson
 sortedcontainers
 typing-extensions
+rook==0.1.160
+distro==1.7.0
+funcsigs==1.0.2
+psutil==5.9.1
```
Comment (nemacysts) [inline requirements-minimal.txt]:
```suggestion
rook
```

this file should only include libraries that are directly imported in production code

**Review by nemacysts — APPROVED** [review]
do we need to do anything to enable rookout for our custom signals? or will they automatically be instrumented by this?

**Review by nemacysts — CHANGES_REQUESTED** [review]
err, misclicked

**Comment by BeryJu** [comment]
> do we need to do anything to enable rookout for our custom signals? or will they automatically be instrumented by this?

Good question, since they are completely independent sub-processes (afaik that's the case), we'll have to import the SDK there too. Also, I can't remember if the signals are continuously running, or if they are started, do their thing and then exit, because if the latter is the case then rookout doesn't work that well with that yet

File: clusterman/tools/rookout.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from os import getenv
+
+
+def enable_rookout():
```
Comment (nemacysts) [inline clusterman/tools/rookout.py]:
```suggestion
def enable_rookout() -> None:
```

dunno how i missed that in the original review :p

**Review by nemacysts — APPROVED** [review]

File: clusterman/tools/rookout.py (hunk 2 above)
Comment (BeryJu) [reply clusterman/tools/rookout.py]: do'h me neither

**Review by nemacysts — APPROVED** [review]

**Comment by BeryJu** [comment]
I'm a bit confused why all the tests succeed since both of the clusterman imports were incorrect (`from clusterman.clusterman.tools.rookout import enable_rookout` instead of `from clusterman.tools.rookout import enable_rookout`)

**Pull request merged**